tests/cypress/tests/standalone/delete_row_test.ts

```diff
@@ -1,8 +1,8 @@
 import DashTable from 'cypress/DashTable';
 
-import { AppMode } from 'demo/AppMode';
+import { ReadWriteModes } from 'demo/AppMode';
 
-Object.values(AppMode).forEach(mode => {
+Object.values(ReadWriteModes).forEach(mode => {
```

Comment:
The readonly case does not allow deleting rows. Use the readwrite modes instead.